lib/tasks/seek_upgrades.rake

```diff
@@ -91,6 +94,26 @@ namespace :seek do
     end
   end
 
+  task(update_morpheus_model: [:environment]) do
+    puts "... updating morpheus model"
+    Model.find_each do |model|
+      next unless model.is_morpheus_supported?
+      begin
+        unless model.model_format
+          model.model_format = ModelFormat.find_by!(title: 'Morpheus')
+        end
+        unless model.recommended_environment
+          model.recommended_environment = RecommendedModelEnvironment.find_by!(title: 'Morpheus')
+        end
+      rescue ActiveRecord::RecordNotFound => e
+        puts "Error: #{e.message}. Ensure that the required 'Morpheus' records exist in the database."
+        abort("Aborting task due to missing 'Morpheus' records.")
+      end
+      model.save
```

Comment:
remove this save, and just use the `update_columns` - this will avoid timestamps being changed. for all those models that have been changed, at the end queue a ReindexingJob with `ReindexingQueue.enqueue(models)`